invokeai/frontend/web/vite.config.mts

```diff
@@ -71,18 +71,18 @@ export default defineConfig(({ mode }) => {
       proxy: {
         // Proxy socket.io to the nodes socketio server
         '/ws/socket.io': {
-          target: 'ws://127.0.0.1:9090',
+          target: 'ws://192.168.5.199:9090',
```

Comment:
Same question here and below in the same file.

Reply:
It's part of local changes, I'll revert them and close related to them comments.